libs/model-assessment/src/lib/ModelAssessmentDashboard/ModelAssessmentEnums.ts

```diff
@@ -2,11 +2,11 @@
 // Licensed under the MIT License.
 
 export enum GlobalTabKeys {
-  DataExplorerTab = "DataExplorerTab",
-  FeatureImportancesTab = "FeatureImportancesTab",
-  ErrorAnalysisTab = "ErrorAnalysisTab",
-  FairnessTab = "FairnessTab",
-  ModelStatisticsTab = "ModelStatisticsTab",
-  CausalAnalysisTab = "CausalAnalysisTab",
-  CounterfactualsTab = "CounterfactualsTab"
+  DataExplorerTab = "Data Explorer",
```

Comment:
Are we display these values anythere?
If yes, must be a bug.
We should not change the value, but display different work from loalization.

Reply:
please check my newest push